ui/app/AppLayouts/Chat/controls/community/HoldingTypes.qml

```diff
@@ -4,4 +4,12 @@ QtObject {
     enum Type {
         Asset, Collectible, Ens
     }
+
+    enum Mode {
```

Comment:
with new enums here name of the file seems to be inappropriate

Reply:
I don't think it is not appropiate. The enums here are defining some `HoldingTypes`, types themselfs, and then actions / modes related to holdings. I would keep it like this. WDYT?